src/aiida/cmdline/commands/cmd_profile.py

```diff
@@ -179,7 +199,7 @@ def profile_show(profile):
 @verdi_profile.command('setdefault', deprecated='Please use `verdi profile set-default` instead.')
 @arguments.PROFILE(required=True, default=None)
 def profile_setdefault(profile):
-    """Set a profile as the default profile (use `verdi profile set-default`)."""
+    """Set a profile as the default profile."""
```

Comment:
Possibly here the hint for `verdi profile set-default` accidentally got lost?

Reply:
I did this intentionally because it was too long for the autogenerated reference in `docs/source/reference/command_line.rst`. The addition of `configure-rabbitmq` pushed the short help text to the right and click imposes a maximum width. The help text became
```
setdefault (Deprecated) Set a profile as the default profile (use `verdi
profile...
```
instead, which I thought would be more confusing than helpful.

Reply:
Ah, alright, I see. Makes sense then!